packages/SwingSet/src/vats/timer/vat-timer.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-use-before-define */
```

Comment:
Notice this line. One consequence of using the singleton pattern for entangled facets, as here, is that there are a lot more use-before-define, to the point that being warned about them is just a noisy distraction.